nav-app/src/app/globals.ts

```diff
@@ -5,5 +5,5 @@
  //
 'use strict';
 
-export const nav_version: string="4.4"
-export const layer_version: string="4.2"
+export const nav_version: string="4.5"
```

Comment:
There's going to be a merge conflict here, we now use package.json to track the Navigator version number.

Reply:
It doesn't look like the package.json and package-lock.json version numbers have been updated with the new version numbers.